lib/jekyll-github-metadata.rb

```diff
@@ -1,4 +1,9 @@
 require 'octokit'
+begin
+  require 'dotenv'
```

Comment:
What do you think about wrapping these two lines (also) in a `if Jekyll.env == 'development'`? 

I believe we'll also need to `require 'jekyll'` then, which I don't *think* will cause any problems.